File: app/src/main/java/fr/scootop/app/player/details/FichePlayerIActivity.java
package fr.scootop.app.player.details;

import androidx.annotation.NonNull;
import androidx.appcompat.app.AppCompatActivity;
import androidx.viewpager.widget.ViewPager;

import android.graphics.Color;
import android.os.Bundle;
import android.os.Handler;
import android.view.Menu;
import android.view.MenuInflater;
import android.view.MenuItem;

import com.github.mikephil.charting.animation.Easing;
import com.github.mikephil.charting.charts.RadarChart;
import com.github.mikephil.charting.components.AxisBase;
import com.github.mikephil.charting.components.Legend;
import com.github.mikephil.charting.components.XAxis;
import com.github.mikephil.charting.components.YAxis;
import com.github.mikephil.charting.data.RadarData;
import com.github.mikephil.charting.data.RadarDataSet;
import com.github.mikephil.charting.data.RadarEntry;
import com.github.mikephil.charting.formatter.IndexAxisValueFormatter;
import com.github.mikephil.charting.interfaces.datasets.IDataSet;
import com.github.mikephil.charting.interfaces.datasets.IRadarDataSet;
import com.google.android.material.tabs.TabLayout;

import java.util.ArrayList;

import fr.scootop.R;
import fr.scootop.app.player.details.fragment.Caracteristique;
import fr.scootop.app.player.details.fragment.EtatCivil;
import fr.scootop.app.player.details.fragment.FicheMain;
import fr.scootop.app.player.details.fragment.Video;

public class FichePlayerIActivity extends AppCompatActivity {

    TabLayout tabLayout;
    ViewPager viewPager;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_fiche_player_iactivity);
        tabLayout = findViewById(R.id.tablayout);
        viewPager = findViewById(R.id.viewpager);
        getTabs();

    }


    public void getTabs(){

        new Handler().post(new Runnable() {
            @Override
            public void run() {

                ViewPagerAdapter viewPagerAdapter = new ViewPagerAdapter(getSupportFragmentManager());

                viewPagerAdapter.addFragment(FicheMain.getInstance(),"Main");
                viewPagerAdapter.addFragment(EtatCivil.getInstance(),"ECV");
                viewPagerAdapter.addFragment(Caracteristique.getInstance(),"Car");
                viewPagerAdapter.addFragment(Video.getInstance(),"Video");

                viewPager.setAdapter(viewPagerAdapter);
                tabLayout.setupWithViewPager(viewPager);

            }
        });
    }
}

File: app/src/main/java/fr/scootop/app/player/details/FichePlayerIActivity.kt
package fr.scootop.app.player.details

import android.os.Bundle
import android.os.Handler
import androidx.appcompat.app.AppCompatActivity
import androidx.viewpager.widget.ViewPager
import com.google.android.material.tabs.TabLayout
import fr.scootop.R
import fr.scootop.app.common.ExtraKey
import fr.scootop.app.player.details.fragment.Caracteristique
import fr.scootop.app.player.details.fragment.EtatCivil
import fr.scootop.app.player.details.fragment.FicheMain
import fr.scootop.app.player.details.fragment.Video
import fr.scootop.data.model.user.domain.Player
import fr.scootop.data.model.user.domain.PlayerSearchResult

class FichePlayerIActivity : AppCompatActivity() {
    var tabLayout: TabLayout? = null
    var viewPager: ViewPager? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_fiche_player_iactivity)
        tabLayout = findViewById(R.id.tablayout)
        viewPager = findViewById(R.id.viewpager)
        getTabs()
        val playerItem = intent.getParcelableExtra<PlayerSearchResult>(ExtraKey.PLAYER_ITEM)
    }

    fun getTabs(): Unit {
        Handler().post {
            val p = Player()
            val viewPagerAdapter = ViewPagerAdapter(supportFragmentManager)

            //Bundle bundle = new Bundle();
            //bundle.putParcelable("player",p);
            viewPagerAdapter.addFragment(FicheMain.getInstance(), "Main")
            viewPagerAdapter.addFragment(EtatCivil.getInstance(), "ECV")
            viewPagerAdapter.addFragment(Caracteristique.getInstance(), "Car")
            viewPagerAdapter.addFragment(Video.getInstance(), "Video")
            viewPager!!.adapter = viewPagerAdapter
            tabLayout!!.setupWithViewPager(viewPager)
        }
    }
}